src/foam/lib/json/Outputter.java

```diff
@@ -111,45 +111,9 @@ public void outputString(String s) {
   }
 
   public String escape(String s) {
-    s = s.replace("\\", "\\\\")
-            .replace("\"", "\\\"")
-            .replace("\t", "\\t")
-            .replace("\r","\\r")
-            .replace("\n","\\n");
-    s = escapeControlCharacters(s);
-    return s;
-  }
-
-  public String escapeMultiline(String s) {
-    s = s.replace("\\", "\\\\");
-    s = escapeControlCharacters(s);
-    return s;
-  }
-
-  public String escapeControlCharacters(String s) {
-    int lastStart = 0;
-    String escapedString = "";
-    char c;
-    for ( int i = 0; i < s.length(); i++ ) {
-      c = s.charAt(i);
-      if ( c >= ' ' ) continue;
-      // Character to hex
-      char right = (char) (c & 0x0F);
-      char left = (char) ((c & 0xF0) >> 4);
-      right += '0';
-      if ( right > '9' ) right += 'A' - '9' - 1;
-      left += '0';
-      if ( left > '9' ) left += 'A' - '9' - 1;
-      char[] escape = new char[] {'\\','u','0','0',left,right};
-      // Add previous string segment
-      escapedString += s.substring(lastStart, i);
-      // Add escape sequence
-      escapedString += new String(escape);
-      lastStart = i + 1;
-    }
-    if ( lastStart != s.length() ) escapedString +=
-      s.substring(lastStart, s.length());
-    return escapedString;
+    StringBuilder sb = new StringBuilder();
```

Comment:
Don't bother with this, since we're just going to replace it with the new code anyway.